lib/components/RoundLinkButton/RoundLinkButton.tsx

```diff
@@ -0,0 +1,58 @@
+import { useId } from 'react';
+import classNames from 'classnames';
+import { PolymorphicComponentProps } from '../../utils/polymorphic';
+
+interface Props {
+  /** Text shown on the link */
+  label: string;
+  /** Selected */
+  selected?: boolean;
+  /** CSS classname */
+  className?: string;
+  /** Link to page/site */
+  href: string;
+}
+
+type RoundLinkButtonProps<C extends React.ElementType> = PolymorphicComponentProps<C, Props>;
+
+export const RoundLinkButton = <C extends React.ElementType = 'a'>({
+  as,
+  label,
+  href,
+  selected = false,
+  className,
+  ...rest
+}: RoundLinkButtonProps<C>) => {
```

Comment:
Do we really need generics?

Reply:
Not necessarily, but with this the component can be `react-router-dom`s `Link` or `NavLink` component. And no need to have router stuff in the design system as dependency.